source/_integrations/dwd_weather_warnings.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: Deutsche Wetter Dienst (DWD) Weather Warnings
+title: Deutscher Wetterdienst (DWD) Weather Warnings
```

Comment:
This value is synced from the codebase. The upstream integration needs to be adjusted to this as well.

Please open up a PR on the code base, and link it to this one.

Reply:
Thanks will do so!